src/cpp/include/openvino/genai/generation_config.hpp

```diff
@@ -72,8 +72,9 @@ enum class StopCriteria { EARLY, HEURISTIC, NEVER };
  * @param rng_seed initializes random generator. Ignored for non continuous batching.
  *
  * Speculative decoding parameters:
```

Comment:
```suggestion
 * Assisting generation parameters:
```
? Similarly to change in generation config